src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php

```diff
@@ -164,6 +164,20 @@ public function testToolbarStylesheetAction()
         $this->assertSame('max-age=600, private', $response->headers->get('Cache-Control'));
     }
 
+    public function testToolbarStylesheetActionFromPath()
+    {
+        $kernel = new WebProfilerBundleKernel();
+        $client = new KernelBrowser($kernel);
+
+        $client->request('GET', '/_wdt/styles');
+
+        $response = $client->getResponse();
+
+        $this->assertSame(200, $response->getStatusCode());
+        $this->assertSame('text/css; charset=UTF-8', $response->headers->get('Content-Type'));
+        $this->assertSame('max-age=600, private', $response->headers->get('Cache-Control'));
+    }
```

Comment:
How is this test different from the existing one?

Reply:
> How is this test different from the existing one?

The other test (above this one) calls the controller directly, this one calls it with a fake request, so that we can check that the route works and has no `.css` extension.

Reply:
So except routing they do the same thing! I think you can improve or replace the other one, this will ease future maintenance or changes :)

Reply:
Done. I don’t know why `charset=UTF-8` is added though.